frontend/src/components/UserForm/UserForm.js

```diff
@@ -0,0 +1,198 @@
+            name="firstName"
+            required
+          />
+        </label>
+        <label className="admin-form-label">
+          Middle Name:
+          <input
+            className="admin-form-input"
+            defaultValue={user?.data?.data?.data?.user.middleName && user.data.data.data.user.middleName}
+            type="text"
+            name="middleName"
+          />
+        </label>
+        <label className="admin-form-label">
+          Last Name:
+          <textarea
+            className="admin-form-input"
+            defaultValue={user?.data?.data?.data?.user.lastName && user.data.data.data.user.lastName}
+            type="text"
+            name="lastName"
+            required
+          />
+        </label>
+        <label className="admin-form-label">
+          Gender:
+          <textarea
+            className="admin-form-input"
+            defaultValue={user?.data?.data?.data?.user.gender && user.data.data.data.user.gender}
+            type="text"
+            name="gender"
+            required
+          />
+        </label>
+        <label className="admin-form-label">
+          Email:
+          <textarea
+            className="admin-form-input"
+            defaultValue={user?.data?.data?.data?.user.email && user.data.data.data.user.email}
+            type="text"
+            name="email"
+            required
+          />
+        </label>
+        <label className="admin-form-label">
+          Admin:
+          <select className="admin-form-input" name="admin" defaultValue={isAdmin.toString()}>
+            <option value="true">True</option>
+            <option value="false">False</option>
+          </select>
+        </label>
+        <label className="admin-form-label">
+          Address1:
+          <textarea
+            className="admin-form-input"
+            defaultValue={user?.data?.data?.data?.user.address1 && user.data.data.data.user.address1}
+            type="text"
+            name="address1"
+            required
+          />
+        </label>
+        <label className="admin-form-label">
+          Address2:
+          <textarea
+            className="admin-form-input"
+            defaultValue={user?.data?.data?.data?.user.address2 && user.data.data.data.user.address2}
+            type="text"
+            name="address2"
+            required
+          />
+        </label>
+        <label className="admin-form-label">
+          City:
+          <textarea
+            className="admin-form-input"
+            defaultValue={user?.data?.data?.data?.user.city && user.data.data.data.user.city}
+            type="text"
+            name="city"
+            required
+          />
+        </label>
+        <label className="admin-form-label">
+          State:
+          <RegionDropdown
+            className="form-input"
+            name="state"
+            country={country}
+            value={region}
+            onChange={(val) => setRegion(val)}
+            required
```

Comment:
There are a few unnecessary `required` statements here. We should drop these as they're not really required. The only thing I think should be required is an email address.